src/hayai_test_result.hpp

```diff
@@ -43,6 +46,54 @@ namespace hayai
 
                 ++runIt;
             }
+
+	    const double mean = RunTimeAverage();
+	    double accu = 0.0;
+
+	    runIt = _runTimes.begin();
+
+	    while (runIt != _runTimes.end())
+            {
+                const uint64_t run = *runIt;
+		const double diff = double(run) - mean;
+		accu += (diff * diff);
+
+                ++runIt;
+            }
+
+	    _timeStdDev = std::sqrt( accu / (_runTimes.size()-1));
```

Comment:
I like to generally have no whitespace with parentheses, but whitespace between operators (`-` etc.)